src/rich-text/commands/index.ts

```diff
@@ -421,6 +474,43 @@ const headingDropdown = (schema: Schema) =>
         )
     );
 
+const overflowDropdown = (schema: Schema, options: CommonViewOptions) =>
+    makeMenuDropdown(
+        "Overflow",
+        _t("commands.overflow"),
+        "overflow-dropdown",
+        () => true,
+        () => false,
+        dropdownItem(
+            _t("commands.tag", { shortcut: getShortcut("Mod-[") }),
+            toggleTagLinkCommand(options.parserFeatures.tagLinks.allowNonAscii),
+            "tag-btn",
+            nodeTypeActive(schema.nodes.tagLink),
+            ["fs-body1", "mt8"]
```

Comment:
Let's remove all these extra classes - I'd prefer if all of our menus looked the same by default. The reason the heading dropdown has these classes is because it is setting the size to visually simulate the differences in headings. I wouldn't mind setting spoiler, sub, sup classes on these, but that'd also set the styling on the shortcuts as well 🤔